app/server/methods.js

```diff
@@ -10,8 +10,16 @@ Meteor.methods({
     this.unblock();
     if (!this.userId) return false;
     
+    console.log(review.text);
```

Comment:
Forgot to hit save.

On Mon, Aug 3, 2015 at 11:24 PM, Oli Evans <notifications@github.com> wrote:

> In app/server/methods.js
> <https://github.com/tableflip/sandstorm-app-market-public/pull/32#discussion_r36138905>
> :
>
> > @@ -10,8 +10,16 @@ Meteor.methods({
> >      this.unblock();
> >      if (!this.userId) return false;
> >
> > +    console.log(review.text);
>
> beep boop. No consoles for the users pls.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/tableflip/sandstorm-app-market-public/pull/32/files#r36138905>
> .
>